be/src/column/const_column.h

```diff
@@ -128,9 +133,19 @@ class ConstColumn final : public ColumnFactory<Column, ConstColumn> {
         }
     }
 
-    void append_default() override { _size++; }
+    void append_default() override {
+        if (_size == 0) {
+            _data->append_default(1);
+        }
+        _size++;
+    }
 
-    void append_default(size_t count) override { _size += count; }
+    void append_default(size_t count) override {
+        if (_size == 0) {
+            _data->append_default(1);
+        }
+        _size += count;
+    }
```

Comment:
reject if data is not emtpy?

Reply:
no, I need append default value on empty column